app/src/main/res/layout/activity_news_story.xml

```diff
@@ -14,9 +14,8 @@
         <ImageView
             android:id="@+id/story_image_view"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:adjustViewBounds="true"
-            android:scaleType="fitXY" />
+            android:layout_height="200dp"
```

Comment:
These two llines can't be deleted, since the image won't be fit for the width of the screen without them.